client/pubsub_test.go

```diff
@@ -40,6 +40,14 @@ func TestPublishEvent(t *testing.T) {
 		assert.NotNil(t, err)
 	})
 
+	t.Run("with metadata", func(t *testing.T) {
+		metadata := map[string]string{
+			"ttlInSeconds": "120",
```

Comment:
Could you please change the metadata to something else so people don't get the idea they should set message expiration this way?